docs/source/advanced/multi_gpu.rst

```diff
@@ -239,6 +239,20 @@ Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
     to be in "exclusive mode", such that only one process at a time can access them.
     For more details see the :doc:`trainer guide <../common/trainer>`.
 
+
+Select Torch Distributed Backend
+--------------------------------
+
+By default, Lightning will choose `nccl` backend over `gloo` when running on GPUs.
+Find more information on [PyTorch supported backends](https://pytorch.org/docs/stable/distributed.html).
+
+Lightning exposes an environment variable ``TORCH_DISTRIBUTED_BACKEND`` for the user to change the backend.
```

Comment:
aren't all env variables in pl prefixed with PL?

Reply:
Added prefix.